tutorials/autotvm/tune_conv2d_cuda.py

```diff
@@ -204,7 +204,7 @@ def conv2d_no_batching(N, H, W, CO, CI, KH, KW, stride, padding):
 # see many error reports. As long as you can see non-zero GFLOPS, it is okay.
 tuner = autotvm.tuner.XGBTuner(task)
 tuner.tune(
-    n_trial=20,
+    n_trial=100,
```

Comment:
probably we don't need this :-)

Reply:
it is removed. I was changing this for benchmarking. Slipping through my finger lol